angr/knowledge_plugins/functions/function_manager.py

```diff
@@ -315,7 +327,7 @@ def function(self, addr=None, name=None, create=False, syscall=False, plt=None):
         :param bool create: Whether to create the function or not if the function does not exist.
         :param bool syscall: True to create the function as a syscall, False otherwise.
         :param bool or None plt: True to find the PLT stub, False to find a non-PLT stub, None to disable this
-                                 restriction.
+                                                         restriction.
```

Comment:
Why changing this line?

Reply:
No reason as far as I can tell. I restore the previous change.